src/styles/Missions.css

```diff
@@ -65,3 +72,9 @@
   border-radius: 5px;
   height: 40px;
```

Comment:
Hi @Oklukeok. 
Kindly reduce the height of the button so that the height of the button you created will stick to the design of the Project requirement.